logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/mapper/CallMapper.kt

```diff
@@ -157,6 +164,13 @@ class CallMapperImpl(
         else -> ConversationEntity.Type.ONE_ON_ONE
     }
 
+    private fun toCallEntityType(conversationType: ConversationType): CallEntity.Type = when (conversationType) {
+        ConversationType.OneOnOne -> CallEntity.Type.ONE_ON_ONE
+        ConversationType.Conference -> CallEntity.Type.CONFERENCE
+        ConversationType.ConferenceMls -> CallEntity.Type.MLS_CONFERENCE
+        ConversationType.Unknown -> CallEntity.Type.CONFERENCE
```

Comment:
Shouldn't here be `CallEntity.Type.UNKNOWN`?

Reply:
Yes! typo